### ISSUE TITLE
There has to be a build as well, because the data is otherwise not displayed

### ISSUE THREAD

**Issue by Idrinth** [hussainnaqvee/github-webhook#4]
https://github.com/hussainnaqvee/github-webhook/blob/2c33f2f914daec354ec5dbb6329e7305816e7d71/index.js#L126

**Comment by hussainnaqvee** [comment]
Yes, I was wondering if I would also concat `&& npm run build` just after running `npm run contributors`. I'll fix that too

**Comment by hussainnaqvee** [comment]
> https://github.com/hussainnaqvee/github-webhook/blob/2c33f2f914daec354ec5dbb6329e7305816e7d71/index.js#L126

Yes, I was wondering if I would also concat && npm run build just after running npm run contributors. I'll fix that too

**Comment by Idrinth** [comment]
> > https://github.com/hussainnaqvee/github-webhook/blob/2c33f2f914daec354ec5dbb6329e7305816e7d71/index.js#L126
> 
> Yes, I was wondering if I would also concat && npm run build just after running npm run contributors. I'll fix that too

run build already contains the contributor fetching :)

**Comment by hussainnaqvee** [comment]
> > > https://github.com/hussainnaqvee/github-webhook/blob/2c33f2f914daec354ec5dbb6329e7305816e7d71/index.js#L126
> > 
> > 
> > Yes, I was wondering if I would also concat && npm run build just after running npm run contributors. I'll fix that too
> 
> run build already contains the contributor fetching :)

I have removed the npm run contributors part as it is redundant. You are right, after building the website, contributors will also be fetched.
